ScreenToGif/Windows/Other/Localization.xaml.cs

```diff
@@ -177,8 +185,8 @@ private void Add_Executed(object sender, ExecutedRoutedEventArgs e)
 
             if (!ofd.FileName.Contains("StringResources"))
             {
-                Dialog.Ok("Action Denied", "The name of file does not follow a valid pattern.",
-                    "Try renaming like (without the []): StringResources.[Language Code].xaml");
+                Dispatcher.Invoke(() => Dialog.Ok("Action Denied", "The name of file does not follow a valid pattern.",
```

Comment:
Is the Dispatcher.Invoke necessary here? I'm not sure, but I believe that the Add_Executed is only called by the command, and the only async code is further down at `await Task.Factory.StartNew(() => LocalizationHelper.ImportStringResource(ofd.FileName));`

Reply:
You are right. I will fix it in the next pull request.